DeviceAdapters/FakeCamera/FakeCamera.h

```diff
@@ -27,7 +27,11 @@
 
 #include "DeviceBase.h"
 
+#ifdef __linux__
+#include <opencv/cv.hpp>
+#else
 #include "opencv/highgui.h"
+#endif

```

Comment:
My guess is that you are using a recent version of OpenCV rather than the ancient version we're currently using (i.e. nothing to do with Linux). This change may therefore break Linux build environments that currently work. More importantly we should probably upgrade OpenCV on all platforms, but that is a harder task. So could you add a comment saying that the `#ifdef __linux__` was added to enable building on Ubuntu 18?